diagnostic.md

```diff
@@ -5,74 +5,90 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+The purpose of a backend is to allow multiple users access to an API to connect them to a single application.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+The model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+The controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+Because the HTML and CSS we create will do everything the views are intended to do.
 ```
 
 What does C.R.U.D stand for?
 
 ```md
-// your response here
+Create Read Update Destroy
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```md
-// your response here
+In the controller
 ```
 
 List at least 5 standard rails actions that C.R.U.D requests correspond to?
 
 ```md
-// your response here
+index
+new
+create
+show
+update
 ```
 
 A user action fires a `GET` request for `/people/1`. Explain in detail each step
 required for data to be returned to the client. (bullet points or ordered list)
 
 ```md
-// your response here
+1. The GET request is recieved and goes through the router and then the controller chooses the best route.
+2. The proper route is sent out with the given paramters assigned to it.
+3.The model performs the request
+4. The model gives a response back
+5. The controller gives back the response returned from the model
+6. The server renders the data as an HTTP response in the browser
 ```
 
 What is the command to generate a new rails-api app?
 
 ```bash
-// your response here
+rails-api new my_api
+new is the command
 ```
 
 What is the command to start an instance of a rails server?
 
 ```bash
-// your response here
+rails server
```

Comment:
This is technically correct, but we want you to use `bundle exec rails server` or `bin/rails server` because these commands are version-specific to the application directory, since it's possible for global versions of Ruby and Ruby on Rails to be different than the local versions.